oracle/types/market.go

```diff
@@ -60,14 +60,18 @@ func ProviderMarketMapFromMarketMap(name string, marketMap mmtypes.MarketMap) (P
 
 // NewProviderMarketMap returns a new provider market map.
 func NewProviderMarketMap(name string, tickerConfigs TickerToProviderConfig) (ProviderMarketMap, error) {
-	if len(tickerConfigs) == 0 {
-		return ProviderMarketMap{}, fmt.Errorf("ticker configs cannot be empty")
-	}
-
 	if len(name) == 0 {
 		return ProviderMarketMap{}, fmt.Errorf("provider name cannot be empty")
 	}
 
+	if len(tickerConfigs) == 0 {
```

Comment:
we should be able to construct the provider even if it has no tickers to support with the given market map.